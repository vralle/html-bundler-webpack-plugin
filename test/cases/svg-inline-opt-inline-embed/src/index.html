<!DOCTYPE html>
<html>
<head>
  <link href="./styles.css" rel="stylesheet">
  <script src="./main.js" defer="defer"></script>
</head>
<body>
  <b>HTML: embed SVG </b>
  <div><img src="@images/icons/home.svg" class="icon"></div>

  <b>HTML: PNG, base64 encoded</b>
  <div><img src="@images/icons/home.png" class="icon"></div>

  <b>CSS: SVG as background image</b>
  <div class="icon-imac"></div>

  <b>JS: SVG import as data URL (default base64)</b>
  <div id="js-svg-data-url"></div>
</body>
</html>