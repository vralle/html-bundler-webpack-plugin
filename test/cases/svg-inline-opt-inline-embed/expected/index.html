<!DOCTYPE html>
<html>
<head>
  <link href="styles.d06f1966.css" rel="stylesheet">
  <script src="main.0c9ecfcd.js" defer="defer"></script>
</head>
<body>
  <b>HTML: embed SVG </b>
  <div><svg viewBox="0 0 16 16" width="16px" height="16px" class="icon"><path d="M 8 1.320313 L 0.660156 8.132813 L 1.339844 8.867188 L 2 8.253906 L 2 14 L 7 14 L 7 9 L 9 9 L 9 14 L 14 14 L 14 8.253906 L 14.660156 8.867188 L 15.339844 8.132813 Z M 8 2.679688 L 13 7.328125 L 13 13 L 10 13 L 10 8 L 6 8 L 6 13 L 3 13 L 3 7.328125 Z"/></svg></div>

  <b>HTML: PNG, base64 encoded</b>
  <div><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAnklEQVR4nN3QIQoCQRjF8Z/CZsEgWAwmg8li9AZewGa1WbyCxWjZ6mUsdotBEPEWKwujLMu6uwoi+ODB8N73/2YYvqxJ8Eea4hacnt/SLIBjjHDFvC68wAXDTDbAGcsqeIUT+gVdD0esi8AGNjigU3JBG3ts0XyEEXZI0Kp6YphJAhOlQRdxCLNKcs53cWCfKhqq0/3bguTFx5V1P9YdrKIzXRZI6v8AAAAASUVORK5CYII=" class="icon"></div>

  <b>CSS: SVG as background image</b>
  <div class="icon-imac"></div>

  <b>JS: SVG import as data URL (default base64)</b>
  <div id="js-svg-data-url"></div>
</body>
</html>